hypothesis-python/src/hypothesis/extra/django/models.py

```diff
@@ -213,6 +213,19 @@ def models(
         raise InvalidArgument(
             u'Missing arguments for mandatory field%s %s for model %s'
             % (u's' if missed else u'', u', '.join(missed), model.__name__))
+
+    primary_key = None
+    for field in result:
+        if model._meta.get_field(field).primary_key:
+            primary_key = field
```

Comment:
You could delete this assignment and use `field` directly below.

Reply:
Well spotted! I cut and pasted without thinking about it...